tone-analyzer/src/main/java/com/ibm/watson/developer_cloud/tone_analyzer/v3/model/ToneChatInput.java

```diff
@@ -0,0 +1,30 @@
+package com.ibm.watson.developer_cloud.tone_analyzer.v3.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+/**
+ * This object represents the input for Tone Chat analysis. It contains a list of utterances.
+ */
+public class ToneChatInput {
```

Comment:
So this field is not the Options for the API, but the input itself. It consists of a list of Utterance. The Utterance Object has a Builder pattern implemented, do you want a Builer pattern for ToneChatInput too?